src/neo/Cryptography/MPT/MPTNode.Branch.cs

```diff
@@ -0,0 +1,56 @@
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Cryptography.MPT
+{
+    public partial class MPTNode : ICloneable<MPTNode>, ISerializable
+    {
+        public const int BranchChildCount = 17;
+        public readonly MPTNode[] Children = new MPTNode[BranchChildCount];
```

Comment:
Now all nodes have 17 children? I don't think it's a good design.

Reply:
Only `NodeType.BranchNode` have 17 children.
I will remove the `new` statement.

Reply:
Done.